src/styles/blocks/main.css

```diff
@@ -0,0 +1,103 @@
+/* #region product-cards */
+
+.product-card {
+  padding: 32px 16px 16px;
+  margin-bottom: 50px;
+  width: 166px;
+  border: 1px solid rgba(229, 229, 229, 1);
+  border-radius: 5px;
+}
+
+.product-card__image {
+  width: 160px;
+  padding: 0 3px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  margin-top: 40px;
+  line-height: 18px;
+  font-weight: 500;
+  color: rgba(6, 11, 53, 1);
+}
+
+.product-card__subtitle {
+  margin-top: 4px;
+  color: rgba(97, 96, 112, 1);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.product-card__rates {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.product-card__price-subtitle {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  height: 16px;
+  margin-right: 4px;
+  width: 16px;
+  background-image: url(./../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars :nth-child(-n + 4) {
+  background-image: url(./../../images/star-active.svg);
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  margin-top: 4px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.product-card__price-block {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.product-card__price {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__buy-link {
+  box-sizing: border-box;
```

Comment:
let's put it unde `*` selector, it must be common behaviour for all your elements